data/transactions/logic/opcodes.go

```diff
@@ -64,7 +64,7 @@ const appAddressAvailableVersion = 7
 
 // EXPERIMENTAL. These should be revisited whenever a new LogiSigVersion is
```

Comment:
It's nitpicking, but I suggest fixing the typos because it can be helpful to find the references via `grep` or IDE search.

```suggestion
// EXPERIMENTAL. These should be revisited whenever a new LogicSigVersion is
```